server/status.go

```diff
@@ -52,10 +51,13 @@ const (
 	// statusLocalStacksKey exposes stack traces of running goroutines.
 	statusLocalStacksKey = statusLocalKeyPrefix + "stacks"
 
-	// statusNodesKeyPrefix exposes status for each of the nodes the cluster.
-	// GETing statusNodesKeyPrefix will list all nodes.
-	// Individual node status can be queried at statusNodesKeyPrefix/NodeID.
-	statusNodesKeyPrefix = statusKeyPrefix + "nodes/"
+	// statusNodesSingleKeyPrefix and statusNodeAllKeyPrefix exposes status for
+	// each of the nodes the cluster.
+	// statusNodesKeyPrefix/nodes -> lists all nodes
+	// statusNodesKeyPrefix/nodes/ -> lists all nodes
+	// statusNodesKeyPrefix/nodes/{NodeID} -> shows only the status for that single node
+	statusNodesSingleKeyPrefix = statusKeyPrefix + "nodes/"
+	statusNodesAllKeyPrefix    = statusKeyPrefix + "nodes"
```

Comment:
Ah, good point.  I'll update that in my next PR which has the same treatment for stores.